roles/ensure_machineset/scripts/reconfigure_machineset.sh

```diff
@@ -14,4 +14,4 @@ echo ${json_input} \
     | jq --arg machinesetname "${machinesetname}" '.spec.selector.matchLabels."machine.openshift.io/cluster-api-machineset" = $machinesetname' \
     | jq -c --arg machinesetname "${machinesetname}" '.spec.template.metadata.labels."machine.openshift.io/cluster-api-machineset" = $machinesetname' \
     | jq -c 'del(.status)|del(.metadata.selfLink)|del(.metadata.uid)' \
-    | jq '.spec.replicas = 1'
+    | jq '.spec.replicas = 0'
```

Comment:
set_scale will later on take care of making sure this value is equal to the requested user value.

reconfigure_machineset's job now is to simply make sure a machineset with that given instance type exists, so 0-replica makes more sense